src/crate/crash/command.py

```diff
@@ -363,6 +364,10 @@ def _verify_connection(self, verbose=False):
             else:
                 results.append(infos + (True, 'OK', ))
 
+        # sort by CONNECTED DESC, SERVER_URL
+        results.sort(key=itemgetter(3), reverse=True)
+        results.sort(key=itemgetter(0))
```

Comment:
doesn't the second sort overwrite the first?

Reply:
Okay, looked like depending on an implementation detail. But the python docs also state it explicitly.